recipes-extended/onl/onl.inc

```diff
@@ -36,6 +36,30 @@ PACKAGE_ARCH = "${MACHINE_ARCH}"
 PROVIDES += "libonlp libonlp-platform libonlp-platform-defaults"
 INSANE_SKIP:${PN} = "file-rdeps"
 
+PACKAGES_DYNAMIC = "^libonlp-.*"
```

Comment:
s/packagroups/packagegroups/ in the commit message.

Reply:
Fixed. For some reason my fingers just don't like this word, I already corrected this a dozen times.